### PR TITLE
feat: awards

### PR DESCRIPTION
**Pull request by totorototo**
add award feature to club

### CODE REVIEW

File: app/routes/screens/clubFeed/ClubFeed.js [hunk 1]
```diff
@@ -90,6 +91,23 @@ class ClubFeed extends Component {
...
           </Card>
+
+          {club.ranking &&
+            <Card
+              dividerStyle={styles.dividerStyle}
+              containerStyle={styles.containerCardStyle}
+              titleStyle={styles.card}
+              title="AWARDS"
+            >
+              {Object.entries(club.ranking).map(([key, value]) =>
```
Comment (guipasmoi) [inline app/routes/screens/clubFeed/ClubFeed.js:102]: can t find any better name than key and value

File: app/store/sagas/clubs.js [hunk 2]
```diff
@@ -19,6 +19,21 @@ import {
...
+function* listActivities(clubID, membersIDs) {
+  const accessToken = yield select(token);
+  const { ids, entities, error } = yield call(
+    listClubActivities,
+    accessToken,
+    clubID
+  );
+  if (!error) {
+    yield put(updateEntity(clubID, "clubs", { activities: ids }));
+    yield put(setEntity("activities", entities));
```
Comment (guipasmoi) [inline app/store/sagas/clubs.js:31]: 2 put you can use reloaddux batch

File: app/store/services/activities.js [hunk 3]
```diff
@@ -95,36 +95,76 @@ export const getAthleteActivities = token => {
   );
 };
 
-export const getRankings = (activities = {}) => {
+export const getRankings = (members = [], activities = {}) => {
```
Comment (guipasmoi) [inline app/store/services/activities.js:98]: i think the body of this function can be done in a smarter way but is it worth the work?

File: app/routes/screens/clubFeed/ClubFeed.js (hunk 1 above)
Comment (totorototo) [reply app/routes/screens/clubFeed/ClubFeed.js:102]: Got your point.

File: app/store/sagas/clubs.js (hunk 2 above)
Comment (totorototo) [reply app/store/sagas/clubs.js:31]: Right. In the same time, we need to document reloaddux api and add few samples.

File: app/store/services/activities.js (hunk 3 above)
Comment (totorototo) [reply app/store/services/activities.js:98]: You are probably right! But the value is not there. I do not think that we spent too much time refactoring this part. We should spend most of our time on reloaddux implementation.

**Pull request merged**